### ISSUE TITLE
Room Editor : Change instance sprite

### ISSUE THREAD

**Issue by Vethergen** [YoYoGames/GameMaker-Bugs#2654]
### Is your feature request related to a problem?

Cannot change sprite of instances.

### Describe the solution you'd like

The ability to change sprites in the double-click window of instances. Or add a reference image that does nothing outside of a visual indication inside the room editor.

**Comment by ThomazDiniz** [comment]
You can already change the sprite by `sprite_index  = _spr;` in creation code, but it would be good to have a visual way of manipulating sprites of instances in the context of the room editor.

This would greatly increase productivity as it would not be necessary to create objects for instances that share the same behavior, but doesn't have the same sprite.

**Comment by tgalluzzo** [comment]
Agreed! This would be of great value for level design in the room editor. You can already change many of the instance properties of an object inside the room editor, for example, you can change the scale of a particular instance, its color, image speed and index. Allowing us to change the sprite of an object instance in the room editor would allow designers to use the same object behavior for different instances that should have different sprites, rather than have to create child objects each time you want to use a new sprite. I've felt the need for this feature come up time and time again.
